test/support/mock/transformations/rotate.ex

```diff
@@ -0,0 +1,17 @@
+defmodule Membrane.VideoCompositor.Mock.Transformations.Rotate do
+  @moduledoc false
+
+  @behaviour Membrane.VideoCompositor.Scene.Transformation
+
+  @enforce_keys [:degrees]
+  defstruct @enforce_keys
+
+  @type t :: %__MODULE__{
+          degrees: non_neg_integer()
+        }
+
+  @impl true
+  def encode(_rotate) do
+    2137
+  end
```

Comment:
...
```suggestion
  @impl true
  def encode(_rotate) do
    69
  end
```

Reply:
absolutely not

Reply:
It is important to stick to our local culture and not import corrupt values and memes from the West.